src/CommunityToolkit.Maui.CameraView/CameraInfo.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Mvvm.ComponentModel;
+using System.Collections.ObjectModel;
+
+#if IOS || MACCATALYST
+using AVFoundation;
+#elif ANDROID
+using AndroidX.Camera.Core;
+#elif WINDOWS
+using Windows.Media.Capture.Frames;
+using Windows.Media.MediaProperties;
+#endif
+
+namespace CommunityToolkit.Maui.Core;
+
+public partial class CameraInfo : ObservableObject
+{
+    public string? Name { get; internal set; }
+    public string? DeviceId { get; internal set; }
+    public CameraPosition? Position { get; internal set; }
```

Comment:
Can these values ever be null? If we are setting them perhaps we can use something other than null and remove the `?`s?

Reply:
Removed nullable